server/stats.go

```diff
@@ -49,6 +87,15 @@ func statsRegisterInt(name string) {
 	expvar.Publish(name, new(expvar.Int))
 }
 
+// Register histogram variable. `bounds` specifies histogram buckets/bins
+// (see comment next to the `histogram` struct definition).
+func statsRegisterHistogram(name string, bounds ...float64) {
```

Comment:
It does not have to be variadic. You could pass the slice itself. The second parameter is always the slice.